test/voting_to_manage_emission_funds_test.js

```diff
@@ -241,6 +247,13 @@ contract('VotingToManageEmissionFunds [all features]', function (accounts) {
         VOTING_START_DATE, VOTING_END_DATE, accounts[5], "memo", {from: votingKey}
       ).should.be.rejectedWith(ERROR_MSG);
     });
+    it('ballot duration must be greater than min ballot duration', async () => {
+      VOTING_START_DATE = moment.utc().add(31 * 60, 'seconds').unix();
+      VOTING_END_DATE = moment.utc().add(31 * 60 + minBallotDuration - 1, 'seconds').unix();
+      await voting.createBallot(
+        VOTING_START_DATE, VOTING_END_DATE, accounts[5], "memo", {from: votingKey}
+      ).should.be.rejectedWith(ERROR_MSG);
```

Comment:
Please add here one more test (right after this line) which would show that the `createBallot` is called successfully for `VOTING_END_DATE = moment.utc().add(31 * 60 + minBallotDuration + 1, 'seconds').unix();`.